.circleci/config.yml

```diff
@@ -652,36 +650,19 @@ jobs:
             - build/test/soltest
             - build/test/tools/solfuzzer
 
-  t_osx_soltest:
-    macos:
-      xcode: "11.0.0"
+  t_osx_soltest: &t_osx_soltest
+    <<: *test_osx
     environment:
       EVM: << pipeline.parameters.evm-version >>
       OPTIMIZE: 0
-      TERM: xterm
     steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - dependencies-osx-{{ checksum ".circleci/osx_install_dependencies.sh" }}
-      - attach_workspace:
-          at: .
       - run: *run_soltest
       - store_test_results: *store_test_results
       - store_artifacts: *artifacts_test_results
 
   t_osx_cli:
-    macos:
-      xcode: "11.0.0"
-    environment:
-      TERM: xterm
+    <<: *test_osx
     steps:
-      - checkout
-      - restore_cache:
-          keys:
-            - dependencies-osx-{{ checksum ".circleci/osx_install_dependencies.sh" }}
-      - attach_workspace:
-          at: .
       - run: *run_cmdline_tests
       - store_artifacts: *artifacts_test_results
```

Comment:
Again, "inherited"  `environment` and `steps` will be overwritten.

Reply:
Oh alright, Noted! Thanks for pointing it out. I'll see to the necessary changes and revert back asap.